lms/extensions/feature_flags/__init__.py

```diff
@@ -176,7 +176,7 @@ def feature(request, feature_flag_name):
 
     def add_feature_flag_providers(_config, *providers):
         """Adapt feature_flags.add_providers()."""
-        providers = [config.maybe_dotted(provider) for provider in providers]
+        providers = [config.maybe_dotted(provider) for provider in providers]  # type:ignore
```

Comment:
I'll add a few ignores, the goal here is to be able no enable `check_untyped_defs`